Sources/Colors/ColorProvider.swift

```diff
@@ -509,33 +555,39 @@ public struct ColorProvider {
     var pillFilterIcon: Color { token.iconInverted }
     var pillSuggestionText: Color { token.text }
     var pillSuggestionBackground: Color {
-        switch Warp.Config.warpTheme {
+        switch Warp.Theme {
         case .finn:
             return Color.dynamicColor(defaultColor: FinnColors.gray200, darkModeColor: FinnColors.gray600)
         case .tori:
             return Color.dynamicColor(defaultColor: ToriColors.gray200, darkModeColor: ToriColors.gray600)
         case .dba:
             return Color.dynamicColor(defaultColor: DBAColors.gray200, darkModeColor: DBAColors.gray600)
+        case .blocket:
+            return Color.dynamicColor(defaultColor: BlocketColors.gray200, darkModeColor: BlocketColors.gray600)
         }
     }
     var pillSuggestionBackgroundHover: Color {
-        switch Warp.Config.warpTheme {
+        switch Warp.Theme {
         case .finn:
             return Color.dynamicColor(defaultColor: FinnColors.gray300, darkModeColor: FinnColors.gray500)
         case .tori:
             return Color.dynamicColor(defaultColor: ToriColors.gray300, darkModeColor: ToriColors.gray500)
         case .dba:
             return Color.dynamicColor(defaultColor: DBAColors.gray300, darkModeColor: DBAColors.gray500)
+        case .blocket:
+            return Color.dynamicColor(defaultColor: BlocketColors.gray300, darkModeColor: BlocketColors.gray500)
         }
     }
     var pillSuggestionBackgroundActive: Color {
-        switch Warp.Config.warpTheme {
+        switch Warp.Theme {
         case .finn:
             return FinnColors.gray400
         case .tori:
             return ToriColors.gray400
         case .dba:
             return DBAColors.gray400
+        case .blocket:
+            return DBAColors.gray400
```

Comment:
Should be `BlocketColors`?
```suggestion
            return BlocketColors.gray400
```

Reply:
Wow thank you for noticing 😅👍